notebooks/process_demetr_results.ipynb

```diff
@@ -0,0 +1,3162 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    id = next(c for c in res_files[0].split('_') if 'id' in c)
</pre><p>Best not to use <code>id</code> as a variable name. Also what this line is doing is quite hard to follow so could be worth a comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/ARC-m4st/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>